### PR TITLE
Added Fields to Employee Table

### PR DESCRIPTION
**Pull request by MathildaShongwe15**

### CODE REVIEW

**Review by Wimmas — APPROVED** [review]

**Review by madwizzard — DISMISSED** [review]

File: RR.UnitOfWork/Entities/HRIS/Employee.cs [hunk 1]
```diff
@@ -53,6 +54,8 @@ public Employee(EmployeeDto employeeDto, EmployeeTypeDto employeeType)
         HouseNo = employeeDto.HouseNo;
         EmergencyContactName = employeeDto.EmergencyContactName;
         EmergencyContactNo = employeeDto.EmergencyContactNo;
+        Active = employeeDto.Active;
+        InActiveReason = employeeDto.InActiveReason;
```
Comment (tbddorothy) [inline RR.UnitOfWork/Entities/HRIS/Employee.cs:58]: Inactive is one word

Comment (tbddorothy) [inline RR.UnitOfWork/Entities/HRIS/Employee.cs:58]:
```suggestion
        InactiveReason = employeeDto.InActiveReason;
```

File: RR.UnitOfWork/Entities/HRIS/Employee.cs [hunk 2]
```diff
@@ -141,6 +144,10 @@ public Employee(EmployeeDto employeeDto, EmployeeTypeDto employeeType)
 
     [Column("emergencyContactNo")] public string? EmergencyContactNo { get; set; }
 
+    [Column("active")] public bool Active { get; set; }
+   
+    [Column("inActive")] public string? InActiveReason{ get; set; }
```
Comment (tbddorothy) [inline RR.UnitOfWork/Entities/HRIS/Employee.cs:149]: Ditto

File: RR.UnitOfWork/Entities/HRIS/Employee.cs [hunk 3]
```diff
@@ -191,7 +198,9 @@ public EmployeeDto ToDto()
             PostalAddress = PostalAddress?.ToDto(),
             HouseNo = HouseNo,
             EmergencyContactName = EmergencyContactName,
-            EmergencyContactNo = EmergencyContactNo
+            EmergencyContactNo = EmergencyContactNo,
+            Active = Active,
+            InActiveReason = InActiveReason
```
Comment (tbddorothy) [inline RR.UnitOfWork/Entities/HRIS/Employee.cs:203]: Ditto

File: RR.UnitOfWork/Entities/HRIS/Employee.cs (hunk 1 above)
Comment (MathildaShongwe15) [reply RR.UnitOfWork/Entities/HRIS/Employee.cs]: Fixed.

File: RR.UnitOfWork/Entities/HRIS/Employee.cs (hunk 2 above)
Comment (MathildaShongwe15) [reply RR.UnitOfWork/Entities/HRIS/Employee.cs]: Fixed.

**Review by madwizzard — APPROVED** [review]

**Review by Wimmas — APPROVED** [review]

**Review by tbddorothy — APPROVED** [review]

**Pull request merged**